### PR TITLE
Add Nolus mainnet

### PR DESCRIPTION
**Pull request by ivivanov**
This PR aims to add Nolus mainnet configurations. 
All IBC channels are tested and work as expected
Pool: 1041 is created usdc/unls

### CODE REVIEW

File: packages/web/config/chain-infos.ts [hunk 1]
```diff
@@ -3598,6 +3598,34 @@ const mainnetChainInfos: SimplifiedChainInfo[] = [
...
+    bip44: {
+      coinType: 118,
+    },
+    bech32Config: Bech32Address.defaultBech32Config("nolus"),
+    currencies: [
+      {
+        coinDenom: "NLS",
+        coinMinimalDenom: "unls",
+        coinDecimals: 6,
+        coinGeckoId: "usd-coin",
```
Comment (JeremyParish69) [inline packages/web/config/chain-infos.ts:3615]: Please correct to "pool:unls"

File: packages/web/config/chain-infos.ts [hunk 2]
```diff
@@ -3598,6 +3598,34 @@ const mainnetChainInfos: SimplifiedChainInfo[] = [
...
+        coinImageUrl: "/tokens/nolus.svg",
```
Comment (JeremyParish69) [inline packages/web/config/chain-infos.ts:3616]: I don't see this image uploaded anywhere

File: packages/web/config/ibc-assets.ts [hunk 3]
```diff
@@ -1813,6 +1813,13 @@ export const IBCAssetInfos: (IBCAsset & {
         destChannelId: "channel-0",
         coinMinimalDenom: "ulore",
       },
+      {
+        counterpartyChainId: "pirin-1",
+        sourceChannelId: "channel-783",
+        destChannelId: "channel-0",
+        coinMinimalDenom: "unls",
+        isVerified: true,
```
Comment (JeremyParish69) [inline packages/web/config/ibc-assets.ts:1821]: I don't recall this ever being approved. Please remove isVerified

File: packages/web/config/chain-infos.ts (hunk 2 above)
Comment (ivivanov) [reply packages/web/config/chain-infos.ts:3616]: It is already there form an old PR
[packages/web/public/tokens/nolus.svg](https://github.com/osmosis-labs/osmosis-frontend/blob/master/packages/web/public/tokens/nolus.svg)

File: packages/web/config/ibc-assets.ts (hunk 3 above)
Comment (ivivanov) [reply packages/web/config/ibc-assets.ts]: removed

File: packages/web/config/chain-infos.ts (hunk 1 above)
Comment (ivivanov) [reply packages/web/config/chain-infos.ts]: Thanks, its fixed now

**Pull request merged**